new-folder-feature-2/new-file-feature2.txt

```diff
@@ -0,0 +1,3 @@
+Created a new file in feature-2 branch
+
+Add another line in feature-2 branch
```

Comment:
Will this code work under the new platform? Have you tested it under the new platform? please confirm this

Reply:
yes it work and i have tested it thoroughly.